src/apiClient.cpp

```diff
@@ -1,5 +1,14 @@
 #include "apiClient.hpp"
 
+#include <memory>
+#include <string>
+
+// project dependencies
+#include "json.hpp"
+
+#define CPPHTTPLIB_OPENSSL_SUPPORT
+#include "httplib.h"
```

Comment:
Note: I think json.hpp and httplib.h should be moved to a folder to make it clear that they are dependencies of the project from other projects that you brought in for this project and don't intend to modify. That will make it easier to see where your project's code begins and ends.

Reply:
Yea I was planning on cleaning up the dep tree soon as well. Good callout.